### PR TITLE
Replace Upcoming Events card with newer content

### LINKED ISSUE #726: Replace "Meet us at EmberFest" sidebar teaser
We should replace the _"Meet us at EmberFest"_ sidebar teaser as the event is now over.

We are currently showing the teaser on these pages:

* https://simplabs.com
* https://simplabs.com/blog/
* https://simplabs.com/expertise/ember/
* https://simplabs.com/why-simplabs/

### TODOs

- [x] figure out what content we want to show in those places instead (e.g. other particular upcoming event, past events (talks), future events (calendar), teaser to the documentary etc.)
- [x] create new sidebar teaser components for the content chosen above and add that on the pages noted above
- [x] potentially remove the current `CardUpcomingEvents` components and associated assets

### PR DESCRIPTION
**Pull request by pichfl**
Drops upcoming events from various pages.

Closes #726

### CODE REVIEW

**Review by marcoow — CHANGES_REQUESTED** [review]
The build fails. We should really replace this with some new content anyway though.

File: src/ui/components/CardEmberSponsors/template.hbs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  @src="/assets/images/card/simplabs-sponsoring-ember.png"
+  @title="Official Ember.js Sponsor"
+  @alt="simplabs :heart: ember"
+>
+  <p>
+    We have been recognized as official
+    <a href="https://emberjs.com/sponsors">
+      sponsors
+    </a>
+    of the Ember.js project along with international brands like LinkedIn and Apple.
```
Comment (marcoow) [inline src/ui/components/CardEmberSponsors/template.hbs]:
```suggestion
    of the Ember.js project along with international brands like LinkedIn and Yahoo.
```

Apple is not a sponsor of the project.

File: src/ui/components/CardEmberSponsors/template.hbs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  @label="Ember.js"
+  @src="/assets/images/card/simplabs-sponsoring-ember.png"
+  @title="Official Ember.js Sponsor"
+  @alt="simplabs :heart: ember"
+>
+  <p>
+    We have been recognized as official
+    <a href="https://emberjs.com/sponsors">
+      sponsors
+    </a>
```
Comment (marcoow) [inline src/ui/components/CardEmberSponsors/template.hbs:11]: The link displays as `block`:

Bildschirmfoto 2019-10-30 um 12 29 02

File: src/ui/components/CardEmberSponsors/template.hbs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<Card
+  @label="Ember.js"
+  @src="/assets/images/card/simplabs-sponsoring-ember.png"
+  @title="Official Ember.js Sponsor"
+  @alt="simplabs :heart: ember"
+>
+  <p>
+    We have been recognized as official
```
Comment (marcoow) [inline src/ui/components/CardEmberSponsors/template.hbs]:
```suggestion
    We recently have been recognized as an official
```

File: src/ui/components/CardEmberSponsors/template.hbs [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    <a href="https://emberjs.com/sponsors">
+      sponsors
```
Comment (marcoow) [inline src/ui/components/CardEmberSponsors/template.hbs]:
```suggestion
      sponsor
```

File: src/ui/components/PageWhySimplabs/template.hbs [hunk 5]
```diff
@@ -59,9 +59,6 @@
       We strongly believe in the value of sharing our expertise and experience with others and are engaged with various conferences and meetups as organizers, speakers and sponsors. Sharing what we do is not only a great learning opportunity for ourselves but allows us to stay up to date with the web community's needs and challenges as well.
     </p>
   </div>
-  <div class="layout.side offset-vertical offset.after-21">
-    <CardUpcomingEvents />
-  </div>
```
Comment (marcoow) [inline src/ui/components/PageWhySimplabs/template.hbs:64]: This breaks the layout:

Bildschirmfoto 2019-10-30 um 12 31 39

Comment (marcoow) [reply src/ui/components/PageWhySimplabs/template.hbs:64]: I guess we could maybe use the playbook sidebar teaser here?

Comment (marcoow) [reply src/ui/components/PageWhySimplabs/template.hbs:64]: Or maybe even better we can create a teaser saying we co-organize EmberFest (which could also resolve #448 )?

**Review by marcoow — CHANGES_REQUESTED** [review]

File: src/ui/components/PageWhySimplabs/template.hbs (hunk 5 above)
Comment (pichfl) [reply src/ui/components/PageWhySimplabs/template.hbs:64]: I like that second idea 👍

**Comment by marcoow** [comment]
@pichfl any progress on this? We still have the _"Meet us at EmberFest"_ on the site, almost 3 weeks after the event.

File: src/ui/components/PageWhySimplabs/template.hbs [hunk 6]
```diff
@@ -60,9 +60,13 @@
     </p>
   </div>
   <div class="layout.side offset-vertical offset.after-21">
-    <CardUpcomingEvents />
+    <Card @label="Conferences" @alt="EmberFest" @src="/assets/images/card/emberfest.svg">
+      <p>
+        We organize EmberFest, the European Community Ember Conference.
```
Comment (marcoow) [inline src/ui/components/PageWhySimplabs/template.hbs]:
```suggestion
        We co-organize EmberFest, the European Community Ember Conference.
```

**Review by marcoow — CHANGES_REQUESTED** [review]
There's one new little thing and one open and yet unaddressed issues from before.

File: src/ui/components/CardEmberSponsors/template.hbs (hunk 1 above)
Comment (pichfl) [reply src/ui/components/CardEmberSponsors/template.hbs]: 🤦‍♂  of course. I was thinking in terms of usage, which do not apply here.

**Review by marcoow — APPROVED** [review]
only CI left to fix

**Pull request merged**